wai-aria/role/role_none_conflict_resolution.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+<head>
+  <title>Role None Conflict Resolution Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!-- none with label(global) on header -->
+<h1 role="none" data-testname="none with global" data-expectedrole="heading" aria-label="x" class="ex">x</h1>
```

Comment:
What does "none with global" mean as the test name? "any tagname with role none and with any global attribute"? If so, can we name the test more explicitly like: 
```suggestion
<h1 role="none" data-testname="heading role none with global attr aria-label" data-expectedrole="heading" aria-label="x" class="ex">x</h1>
```